.github/workflows/ci.yml

```
@@ -0,0 +1,110 @@
+name: Continous Integration
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+
+env:
+  CI: true
+  NODE_OPTIONS: --max-old-space-size=4096
+
+jobs:
+  test:
+    name: Test
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [14, 15]
+        mongodb-version: [4.4]
+        redis-version: [6]
+        include:
+          - os: windows-latest
+            node-version: 15
+          - os: macos-latest
+            node-version: 15
```

Can we test all supported versions of Node.js for the Ubuntu runner, and use v14 (latest LTS) for the Windows and macOS runners?

```suggestion
        os: [ubuntu-latest]
        node-version: [10, 12, 14, 15]
        mongodb-version: [4.4]
        redis-version: [6]
        include:
          - os: windows-latest
            node-version: 14
          - os: macos-latest
            node-version: 14
```

---

Applied